### ISSUE TITLE
OpenAPI v2 annotations

### ISSUE THREAD

**Issue by saeid-ir** [rules-proto-grpc/rules_proto_grpc#93]
Hi,

As the example and repository files show, there is nothing mentioned about `protoc-gen-openapiv2/options/annotations.proto` exist in in `grpc-gateway` repository.
is there any example on how to use this annotation file for generation swagger definitations?

**Comment by aaliddell** [comment]
I think you should be able to use `@grpc_ecosystem_grpc_gateway//protoc-gen-openapiv2/options:options_proto` as a dependency to the `proto_library`. Let me know if that works.

**Comment by saeidakbari** [comment]
No, it doesn't work.
```
no such package '@grpc_ecosystem_grpc_gateway//protoc-gen-openapiv2/options': BUILD file not found in directory 'protoc-gen-openapiv2/options' of external repository @grpc_ecosystem_grpc_gateway. Add a BUILD file to a directory to mark it as a package.
```
I add it to `deps` section in `proto_library`

**Comment by aaliddell** [comment]
Possibly fixed by #98, where grpc-gateway version is updated from 1.x.x to 2.x.x
